cmd/snaptel/config.go

```diff
@@ -40,6 +41,7 @@ type restAPIConfig struct {
 }
 
 func (c *config) loadConfig(path string) error {
+	log.Warning("The snaptel configuration file will be deprecated. Find more information here: https://intelsdi.atlassian.net/browse/SDI-2580 ")
```

Comment:
If you change this log message, what do you think about something similar to others existing (e.g. https://github.com/intelsdi-x/snap/blob/master/control/available_plugin.go#L103). So it might look like below:
```
log.WithFields(log.Fields{
   "_module":     "snaptel-config",
   "_block":      "loadConfig",
   "config_path": path, 
}).Warning("The snaptel configuration file will be deprecated. Find more information here: <GH issue>")
```
<sup>*</sup>where GH issue is an announcement of depreciation loading config by snaptel - is there any already open?